public/router.js

```diff
@@ -41,8 +41,19 @@ function clearErrorBar() {
     store.dispatch(clearError());
 }
 
+function onRouterUpdate() {
+  const domainMatch = /^.*\.(?<environment>local|code)\.dev-gutools\.co\.uk$|^.*\.gutools\.co\.uk$/
+      .exec(location.hostname);
+  if (domainMatch) {
+    const stage = (domainMatch.groups?.environment || "PROD");
+    const telemetryUrl = stage === "PROD" ? "user-telemetry.gutools.co.uk" : `user-telemetry.${stage}.dev-gutools.co.uk`
+    const image = new Image();
+    image.src = `https://${telemetryUrl}/guardian-tool-accessed?app=tag-manager&stage=${stage.toUpperCase()}&path=${window.location.pathname}`;
```

Comment:
Just curious, I'm guessing this is just firing a GET request to the url, so could `fetch()` be used instead of an image? I'd guess there aren't any CORS or security issues.